src/main/java/net/liquidpineapple/pang/objects/LifeSystem.java

```diff
@@ -38,21 +51,32 @@ public void updateLifes() {
                     break;
                 case 0:
                     changeImage(textureH0);
-                    System.exit(0);
+                    Application.getBoard().changeScreen(new MainMenu());
                     break;
             }
             lastLives = lives;
         }
     }
 
-
+    /**
+     * Method that increases the life with 1 if needed.
+     */
     public static void gainLife() {
         if (lives < 3) {
             lives += 1;
         }
     }
 
+    /**
+     * Method that returns the current ammount of lives.
+     * @return - the remaining ammount of lives.
+     */
```

Comment:
Could you convert this whole method to the lombok `@Getter` on the field (I know you didn't write this method, but you're changing some things anyway)